syntax = "proto3";

package krakend;

service KrakenDService {
  rpc ForwardRequest (Request) returns (Response) {}
}

message Request {
  string method = 1;
  string path = 2;
  map<string, string> headers = 3;
  bytes body = 4;
}

message Response {
  int32 status_code = 1;
  map<string, string> headers = 2;
  bytes body = 3;
} 